gdpr/tests/snapshots/snap_test_gdpr_api.py

```diff
@@ -681,6 +681,7 @@
                                     "children": [],
                                     "key": "FREE_SPOT_NOTIFICATION_SUBSCRIPTIONS",
                                 },
+                                {"key": "NOTES", "value": ""},
```

Comment:
Trivia: there could be a value that we test.

Reply:
Taken into account in 8a28302cd15c3354118ed4bec81ce39559adadfb by the [force push](https://github.com/City-of-Helsinki/kukkuu/compare/93256b48babb5e17bc8336791d7b77e475471f2a..8a28302cd15c3354118ed4bec81ce39559adadfb)